test/unit/foreman/renderer/scope/macros/host_template_test.rb

```diff
@@ -172,7 +172,7 @@ class HostTemplateTest < ActiveSupport::TestCase
 
     test 'grub_pass helper returns the grub password if enabled' do
       @scope.instance_variable_set('@host', host)
-      @scope.instance_variable_set('@grub', true)
+      @scope.expects(:host_param).with('encrypt_grub').returns(true)
       assert_equal "--iscrypted --password=#{host.grub_pass}", @scope.grub_pass
     end
   end
```

Comment:
Also a negative test would be nice to have to make sure we do not accidentally encrypt all users grub loaders :-)